schema/deploy/shipping/views.sql

```diff
@@ -728,4 +728,53 @@ comment on view shipping.return_results_v2 is
     'Version 2 of view of barcodes and presence/absence results for return of results on website';
 
 
+create or replace view shipping.hcov19_observation_with_presence_absence_result_v1 as
+
+select
+        presence_absence.created::date as sample_tested_date,
+        present as test_result,
+        sample.details->>'sample_origin' as sample_manifest_origin,
+        best_available_encounter_date,
+        age_range_fine,
+        sex,
+        residence_puma,
+        best_available_site,
+        individual,
+        sample.identifier as sample,
+        symptoms,
+        best_available_site_type
+
+    from shipping.sample_with_best_available_encounter_data_v1
+    join warehouse.sample using (sample_id)
+    full outer join shipping.incidence_model_observation_v3 on
+      (sample.identifier = incidence_model_observation_v3.sample)
+    left join warehouse.presence_absence using (sample_id)
+    left join warehouse.target using (target_id)
+    left join warehouse.organism using (organism_id)
+    -- TODO we'll replace this via a replacement join on
+    -- incidence_model_observation_v4 with PUMAs once it's merged to master
+    left join (
+      select
+        encounter_id,
+        tract.hierarchy->'puma' as residence_puma,
+        tract.hierarchy->'neighborhood_district' as residence_neighborhood_district
+      from warehouse.encounter_location
+      left join warehouse.location as address using (location_id)
+      left join warehouse.tract as tract on (address.hierarchy -> 'tract') = tract.identifier
+      where relation = 'residence'
+      or relation = 'lodging'
+    ) as residence using (encounter_id)
+    where (
+        lineage ~ 'Human_coronavirus.2019'
+        or best_available_encounter_date >= '2020-03-01'::date
+      )
+      and sample.details->>'sample_origin' != 'es'
+      and not control
+    ;
+
+
+comment on view shipping.hcov19_observation_with_presence_absence_result_v1 is
+  'Custom view similar to shipping.observation_with_presence_absence_result_v2 but for hCoV-19 positive samples';
+
+
```

Comment:
Not sure what kind of grants I should assign here given the new RLS. 
Maybe this?
```
revoke all
    on shipping.hcov19_observation_with_presence_absence_result_v1
  from "hcov19-visibility";

grant select
   on shipping.hcov19_observation_with_presence_absence_result_v1
   to "hcov19-visibility";
```

Reply:
Or perhaps a revoke all from public?

Reply:
This view will need the same grants/revokes as the reportable conditions view.